{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/urls.py

```diff
@@ -31,3 +31,9 @@
 ]
 
 {%- endif %}
+
+{% if cookiecutter.use_analytics == 'y' -%}
```

Comment:
you need to put this before the wagtail catchall, or it will never be hit because the wagtail regex is greedy